include/eve/memory/aligned_allocator.hpp

```diff
@@ -72,11 +74,13 @@ namespace eve
 
     void * allocate_aligned(std::size_t n, std::size_t a)
     {
+      if (a < 8U) a = 8U;
```

Comment:
`a = std::min(a,8U);`  ?

Reply:
max. I thought about it but I figured I can not bring algorithm. 

Also size_t and U are different types on apple clang - I'd have to cast

Reply:
Tried - because of the cast I don't want to.